pkg/cmd/openshift-sdn-cni/openshift-sdn.go

```diff
@@ -304,6 +304,13 @@ func (p *cniPlugin) CmdDel(args *skel.CmdArgs) error {
 	return err
 }
 
+func (p *cniPlugin) CmdCheck(args *skel.CmdArgs) error {
```

Comment:
Here I included the same behaviour (and comment) we have in ovnk:
https://github.com/ovn-org/ovn-kubernetes/blob/5a08eb28c1618ce9748073b42bd9ef6c795e7a98/go-controller/pkg/cni/cni.go#L241